src/en/charms-storage.md

```diff
@@ -1,4 +1,5 @@
-Title: Using storage with Juju charms
+Title: Using Juju Storage
+TODO:  bug tracking: https://pad.lv/1708212
 
 # Using Juju Storage
```

Comment:
Good question. I just changed the page 'Title:' according to the existing top header.